README.md

```diff
@@ -65,6 +65,13 @@ brew install grype
 
 **Note**: Currently, Grype is built only for macOS and Linux.
 
+### MacPorts
+
+On macOS, Grype can additionally be installed via [MacPorts](https://www.macports.org):
```

Comment:
Since this is not an official package (and not signed) we should denote this:
```suggestion
On macOS, Grype can additionally be installed from the [community maintained MacPort](https://ports.macports.org/port/grype/):
```